index.js

```diff
@@ -72,10 +79,14 @@ module.exports = function shortcode_plugin (md, shortcodes) {
 
     function render (tokens, idx, options, env, self) {
         const content = tokens[idx].content;
-
+        
         let parameters = {}, attr;
         while (attr = reAttrs.exec(content)) {
-            if (attr[3]) { //quoted
+            if (attr[4]) { //interpolated
+                let expr = attr[4].slice(1, -1).trim();
+                let interpolatedResult = interpolator(expr, env);
```

Comment:
This line is redundant.

Reply:
Thanks. That was left over from debugging. I'll remove it.